tests/main_test.ts

```diff
@@ -841,6 +841,27 @@ Deno.test("PORT environment variable", async () => {
   for await (const _ of lines) { /* noop */ }
 });
 
+Deno.test("HOST environment variable", async () => {
```

Comment:
I see your comment about testing port, but since Fresh relies on `Deno.serve`, I guess there aren't any tests for testing hostname/port from the options bag. We can assume it works from upstream.

Perhaps you could modify this to also set the port via an environment variable?

Reply:
Add the PORT to this test. Thank you!